funcr/funcr_test.go

```diff
@@ -1361,3 +1361,35 @@ func TestOptionsTimestampFormat(t *testing.T) {
 		t.Errorf("\nexpected %q\n     got %q", expect, capt.log)
 	}
 }
+
+func TestOptionsLogInfoLevel(t *testing.T) {
+	testCases := []struct {
```

Comment:
can you please add the case of `nil` (you will need to change it to a pointer in the table input) to prove that "level" is the result?  We have a helpful `ptrstr()` function to make this easy.